### DISCUSSION TITLE
Defaults for PPL values?

### DISCUSSION THREAD

**Discussion by donn** [The-OpenROAD-Project/OpenROAD#4890] [Q&A] [answered]
For these flags in PPL:

* `set_pin_length -hor_length`
* `set_pin_length -ver_length`
* `place_pins -min_distance`

It isn't super clear from the documentation what the default values are/where they would be obtained from otherwise. Does anyone have any idea?

**Comment by maliberty** [comment]
@eder-matheus please respond when you can

**Comment by eder-matheus** [comment, marked as answer]
@donn I created this PR https://github.com/The-OpenROAD-Project/OpenROAD/pull/4891 updating the documentation with this data. In summary, the default length of the pins is the minimum necessary to respect the minimum area of the layer a pin was placed, considering that the width of a pin is the min width of the layer the pin was placed.

About the min distance, the default is the length of two tracks between each pin.
